src/foam/dao/EasyDAO.js

```diff
@@ -61,7 +61,11 @@ foam.CLASS({
     'foam.dao.RequestResponseClientDAO',
     'foam.dao.SequenceNumberDAO',
     'foam.dao.SyncDAO',
-    'foam.dao.TimingDAO'
+    'foam.dao.TimingDAO',
+    {
+      path: 'foam.dao.JournalType',
+      flags: ['java']
```

Comment:
Any reason why we have to flag this for only java?  Would be better to just switch to JournalType for both JS and Java.

Reply:
Since journals are only being used on the server side, and we are trying to limit the amount of JS being served to the client. I thought this is what we wanted.

Reply:
You're not going to save a lot of code by stripping this requirement.  Basically just the size of the JournalType .js file will be saved.  But at the cost of decreased understandability of the codebase.  Consistency between platforms aids in understandability, so better to keep them consistent when we can.

Reply:
Removed